exec/cfg.c

```diff
@@ -49,6 +49,7 @@
 #include <limits.h>
 #include <errno.h>
 #include <string.h>
+#include <alloca.h>
```

Comment:
you probably already saw this, but BSD doesn´t have alloca.h. We probably need to add a check in configure.ac and if the header exists, then #include it. Same for cfg.c down below.